.github/workflows/cmake.yml

```diff
@@ -81,7 +81,7 @@ jobs:
             - name: Test executing of the grass command
               run: .github/workflows/test_simple.sh
             - name: Run tests
-              run: .github/workflows/test_thorough.sh --config .gunittest.cfg --min-success 72
+              run: .github/workflows/test_thorough.sh --config .gunittest.cfg --min-success 73
```

Comment:
Not higher?

Reply:
Now after rebase, yes indeed.

Reply:
Ah, the percentage is displayed rounded, but needs to be an int in the arg parse. So, since 294/298= 0.986577, we need to use 98 instead of 99.